content/docs/dvclive/index.md

```diff
@@ -42,6 +42,15 @@ live.log_artifact("model.pt")
 
 See `Live.log_artifact()`.
 
+</tab>
+<tab title="Models">
+
+```python
+live.log_artifact("model.pt", type="model")
```

Comment:
No strong opinion, but I guess you decided not to suggest explicit names?

Reply:
Right, missed that. Let's suggest it.